server/api/settings.py

```diff
@@ -101,7 +102,7 @@
         "NAME": os.environ.get("POSTGRES_NAME") or "postgres",
         "USER": os.environ.get("POSTGRES_USER") or "postgres",
         "PASSWORD": os.environ.get("POSTGRES_PASSWORD") or "password",
-        "HOST": os.environ.get("POSTGRES_HOST") or "host.docker.internal",
+        "HOST": os.environ.get("POSTGRES_HOST") or "localhost",
```

Comment:
Instead of modifying the shared settings.py, please add variable to your own .env

```
POSTGRES_HOST=localhost
```

```suggestion
        "HOST": os.environ.get("POSTGRES_HOST") or "host.docker.internal",
```

Reply:
Thanks for your suggestion!